ngraph/python/src/pyngraph/frontend_manager.cpp

```diff
@@ -101,3 +103,38 @@ void regclass_pyngraph_FEC(py::module m)
            const ngraph::frontend::FrontEndCapabilities& b) { return a == b; },
         py::is_operator());
 }
+
+void regclass_pyngraph_ErrorCode(py::module m)
+{
+    py::class_<ngraph::frontend::ErrorCode, std::shared_ptr<ngraph::frontend::ErrorCode>> type(
+        m, "ErrorCode");
```

Comment:
Can it be some 'frontend-specific'? Like "FrontEndErrorCode" or "from ngraph.frontend import ErrorCode"

Reply:
sure

Reply:
done